requirements-test.txt

```diff
@@ -2,7 +2,7 @@
 pep8
 coverage
 mock
-pytest >=4.6
+pytest >=4.6,<6
```

Comment:
@gdubicki why this change?

Reply:
As I have written in the commit message of this change: because our tests are incompatible with Pytest 6. Perhaps I should have elaborated on that more, let me do that here. :)

I have squashed my commits so you won't find the link to a failed test in this PR, but after a while of digging in travis-ci.org/github/voxpupuli/puppetboard I found it here: https://travis-ci.org/github/voxpupuli/puppetboard/builds/755852596

In the output we have:
```
________________________ ERROR collecting test session _________________________

Direct construction of Pep8Item has been deprecated, please use Pep8Item.from_parent.

See https://docs.pytest.org/en/stable/deprecations.html#node-construction-changed-to-node-from-parent for more details.
```
...which, as explained under that URL, is something that:
```
[has] Changed in version 6.0. [of Pytest]
```

I would prefer to not update the tests for Pytest 6 in this PR. Firstly it is out of its scope. Secondly I am afraid that starting to work on in here would delay merging this PR to a point where it will start to be out of date or have to resolve conflicts that will arise.